test/image/mocks/zindex-contour.json

```
@@ -0,0 +1,19 @@
+{
+"data": [
+    {
+        "type": "scatter",
+        "name": "scatter",
+        "y": [-1,0,1,2,3],
+        "x": [-1,0,1,2,3],
+        "zindex": 1
+    }, {
+        "type": "contour",
+        "z": [
+            [1, 2, 3],
+            [2, 0, 0],
+            [3, 0, 3]
+        ],
+        "zindex": 10
+    }
+]
```

Let's provide width and height here: 
```suggestion
 ], { "width": 400, "height": 400" }
```